vms/evm/customlogs/log_ext.go

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2019-2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package customlogs
+
+import ethtypes "github.com/ava-labs/libevm/core/types"
```

Comment:
```suggestion
import "github.com/ava-labs/libevm/core/types"
```

nit: no need for package alias [[ref](https://github.com/uber-go/guide/blob/master/style.md#import-aliasing)]